Sources/BrightroomEngine/Engine/ImageTool.swift

```diff
@@ -175,8 +291,9 @@ public enum ImageTool {
       height: sourceImage.height
     )
 
-    let targetSize = imageSize.scaled(maxPixelSize: maxPixelSize)
-
+    let targetSize: CGSize = {
+      imageSize.scaled(maxPixelSize: maxPixelSize)
+    }()
```

Comment:
@ntnmrndn 
any reason? but I don't care

Reply:
Not anymore. I edited it